core-feature-pack/src/main/resources/content/docs/contrib/scripts/init.d/wildfly-init-debian.sh

```diff
@@ -142,7 +142,7 @@ fi
 export JBOSS_CONSOLE_LOG
 
 # Location to set the pid file
-JBOSS_PIDFILE="/var/run/$NAME/$NAME.pid"
+JBOSS_PIDFILE="/var/run/$NAME/wildfly.pid"
```

Comment:
Note to self: I don't think this is directly part of WFCORE-1604 but it makes this script consistent with wildfly-init-redhat.sh which seems like a good thing.